airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/debezium/CdcSourceTest.java

```diff
@@ -84,11 +84,11 @@ public abstract class CdcSourceTest<S extends Source, T extends TestDatabase<?,
   protected T testdb;
 
   protected String createTableSqlFmt() {
-    return "CREATE TABLE %s.%s(%s);";
+    return "CREATE TABLE `%s`.`%s`(%s);";
```

Comment:
uh? back ticks? I doubt this is an universally working change. If it is, it probably deserve a comment with a reference.

Reply:
yeah i think you are right. I'll push this change to a lower level and apply it only to mysql